website/templates/project/contributors.mako

```diff
@@ -366,11 +406,26 @@
         <td>
             <div class="header" data-bind="visible: accessRequest.expanded()  && $root.collapsed()"></div>
             <div class="td-content" data-bind="visible: !$root.collapsed() || accessRequest.expanded()">
+            <div data-bind="ifnot: accessRequest.user.is_institutional_admin">
```

Comment:
Is this correct, or is it that the request was made from the institutional dashboard? Does the request need to store this info, rather than inferring it from the institutional admin status? This goes back to not wanting all institutional admins to be automatically classified as a curator request.

Reply:
Yes you are correct, I checked with Product again and they confirmed. This should ensure the user actually has made a pending institutional access request, not a normal access request.